TAO/TAO_IDL/include/ast_expression.h

```diff
@@ -335,12 +344,14 @@ class TAO_IDL_FE_Export AST_Expression
   AST_Param_Holder *param_holder_;
   // Non-zero if we were created from a reference template param.
 
-private:
```

Comment:
Why remove several privates in multiple headers? Should all be public?

Reply:
I made a few things public or protected because I needed them, but the most should have been because they were redundant. This one might have been a mistake though, I'll have to check.

Reply:
Nevermind, I wasn't reading Github right, this is redundant as well.